test/option/debug-memoizer.py

```diff
@@ -51,36 +48,35 @@ def cat(target, source, env):
 # names in the implementation, so if we change them, we'll have to
 # change this test...
 expect = [
-    "Memoizer (memory cache) hits and misses",
-    "Base.stat()",
+    # "Memoizer (memory cache) hits and misses",
```

Comment:
This one commented out because it prints even if the memoizer failed to initialize, since the actual heading/print-when-done is handled by the normal option processing, not the special up-front processing. Since the match requires "at least one of" it would still match and obscure the failure in the combined-debug-option case.